### PR TITLE
deprecate default alchemy key (#955)

### PR DESCRIPTION
**Pull request by technophile-04**

### CODE REVIEW

**Comment by technophile-04** [comment]
Note we shouldn't squash and merge this, instead of a merge commit. 

Also planning to release new version after we merge this PR since current latest version is broken :/

**Review by rin-st — APPROVED** [review]
Lgtm!

File: .changeset/happy-books-collect.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"create-eth": patch
+---
+
+depreciate default alchemy key (scaffold-eth#955)
```
Comment (rin-st) [inline .changeset/happy-books-collect.md:5]: deprecate*

**Comment by technophile-04** [comment]
Thanks @rin-st! Lol sooo fast!!

**Pull request merged**